tests/fixtures/client.py

```diff
@@ -0,0 +1,75 @@
+import asyncio
```

Comment:
Need high-level commenting/documentation explaining the test framework, fixture setup